src/tinyspline.c

```diff
@@ -1,6 +1,9 @@
 #define TINYSPLINE_EXPORT
 #include "tinyspline.h"
+
+#if !defined(TINYSPLINE_NO_SERIALIZATION)
 #include "parson.h" /* serialization */
```

Comment:
`#include <stdio.h>` can also be excluded as we only use `FILE` and `fopen` from it. I propose to move this line into the `!defined(TINYSPLINE_NO_SERIALIZATION)` block.